pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -214,6 +232,36 @@ func (m *manager) Run(ctx context.Context) {
 	}()
 }
 
+// Gernerate pod resize completed event message
+func (m *manager) podResizeCompletionMsg(allocatedPod *v1.Pod) string {
+	podResizeSource := &podResourceSummary{}
+	podutil.VisitContainers(&allocatedPod.Spec, podutil.InitContainers|podutil.Containers,
+		func(allocatedContainer *v1.Container, containerType podutil.ContainerType) bool {
+			allocatedResources, exists := m.GetContainerResourceAllocation(allocatedPod.UID, allocatedContainer.Name)
+			if exists {
+				allocation := containerAllocation{
+					Name:      allocatedContainer.Name,
+					Resources: allocatedResources,
+				}
+				switch containerType {
+				case podutil.InitContainers:
+					podResizeSource.InitContainers = append(podResizeSource.InitContainers, allocation)
+				case podutil.Containers:
+					podResizeSource.Containers = append(podResizeSource.Containers, allocation)
+				}
+			}
+			return true
+		})
+
+	podResizeMsgDetailsJSON, err := json.Marshal(podResizeSource)
+	if err != nil {
+		klog.ErrorS(err, "Failed to serialize resource summary", "pod", format.Pod(allocatedPod))
+		return "Pod resize completed: "
```

Comment:
```suggestion
		return "Pod resize completed"
```

why the `:`? Are we expecting something to be appended to this in the error case?

Reply:
No, you are correct, I deleted it